sdk/ai/azure-ai-inference/tests/test_model_inference_async_client.py

```diff
@@ -677,3 +691,73 @@ async def test_async_embeddings_with_auth_failure(self, **kwargs):
             assert "auth token validation failed" in e.message.lower()
         await client.close()
         assert exception_caught
+
+    # **********************************************************************************
+    #
+    #                            TRACING TESTS - CHAT COMPLETIONS
+    #
+    # **********************************************************************************
+
+    def setup_memory_trace_exporter(self) -> MemoryTraceExporter:
+        # Setup Azure Core settings to use OpenTelemetry tracing
+        settings.tracing_implementation = "OpenTelemetry"
+        trace.set_tracer_provider(TracerProvider())
+        tracer = trace.get_tracer(__name__)
+        memoryExporter = MemoryTraceExporter()
+        span_processor = SimpleSpanProcessor(memoryExporter)
+        trace.get_tracer_provider().add_span_processor(span_processor)
+        return span_processor, memoryExporter
+
+    def modify_env_var(self, name, new_value):
+        current_value = os.getenv(name)
+        os.environ[name] = new_value
+        return current_value
+
+    @ServicePreparerChatCompletions()
+    @recorded_by_proxy_async
+    async def test_chat_completion_async_tracing_content_recording_disabled(self, **kwargs):
+        # Make sure code is not instrumented due to a previous test exception
+        try:
+            AIInferenceInstrumentor().uninstrument()
+        except RuntimeError as e:
+            pass
+        self.modify_env_var(CONTENT_TRACING_ENV_VARIABLE, "False")
+        client = self._create_async_chat_client(**kwargs)
+        processor, exporter = self.setup_memory_trace_exporter()
+        AIInferenceInstrumentor().instrument()
+        response = await client.complete(
+            messages=[
+                sdk.models.SystemMessage(content="You are a helpful assistant."),
+                sdk.models.UserMessage(content="What is the capital of France?"),
+            ],
+        )
+        processor.force_flush()
+        spans = exporter.get_spans_by_name_starts_with("chat ")
+        if len(spans) == 0:
+            spans = exporter.get_spans_by_name("chat")
```

Comment:
span name should be `{operation} {model}`, e.g. `chat gpt-4` if model is available at request time and just `chat` if model is not available

Reply:
yes and model is not available in these cases